pkg/webhook/webhook.go

```diff
@@ -25,6 +25,11 @@ const (
 	logFieldError          = "error"
 )
 
+var (
+	errClientMustProvideContentType  = fmt.Errorf("client must provide a content type")
```

Comment:
errors.New() is more appropriate if no formatting is required, but no big deal there.